src/Functions/map.cpp

```diff
@@ -138,17 +138,16 @@ class FunctionMap : public IFunction
     }
 };
 
-struct NameMapContains { static constexpr auto name = "mapContains"; };
```

Comment:
@alexey-milovidov I think we cannot remove this line, because line199 needs this struct to call  `FunctionArrayIndex.executeImpl`.